dashboard/lib/devise/models/custom_timeoutable.rb

```diff
@@ -7,13 +7,13 @@ module Models
     module CustomTimeoutable
       extend ActiveSupport::Concern
 
-      included do
-        devise :timeoutable
-      end
-
       # Devise provides the timeout_in method. Override this functionality here
       # to set custom session timeout values for different users.
       def timeout_in
+        if Policies::Lti.restricted_user?(self)
+          return 24.hours
+        end
+
         self.class.timeout_in
```

Comment:
NIT: can you call `super` here?

Reply:
I think you can, as long as you are including this concern after the devise concern is included, which is now happening.